doc/classes/Object.xml

```diff
@@ -136,7 +136,7 @@
 				        }
 				    }
 
-				    private List&lt;int&gt; _numbers = new();
+				    private Array&lt;int&gt; _numbers = new();
```

Comment:
We fully-qualify the collection names in the documentation to avoid confusion with similarly named types from the BCL.

```suggestion
				    private Godot.Collections.Array&lt;int&gt; _numbers = new();
```

Reply:
That makes sense! Thanks for the patch.